bundles/org.openhab.binding.guntamatic/README.md

```diff
@@ -162,8 +162,8 @@ Example list of Channels using a Guntamatic Biostar 15kW Pellets Heating System
 | `076-program-hc6`          | Program HC6            | `String`               |      | 🔓 W0                 | R/O      | false    |
 | `077-program-hc7`          | Program HC7            | `String`               |      | 🔓 W0                 | R/O      | false    |
 | `078-program-hc8`          | Program HC8            | `String`               |      | 🔓 W0                 | R/O      | false    |
-| `079-interuption-0`        | Interuption 0          | `String`               |      | 🔓 W0                 | R/O      | false    |
-| `080-interuption-1`        | Interuption 1          | `String`               |      | 🔓 W0                 | R/O      | false    |
+| `079-interruption-0`       | Interruption 0         | `String`               |      | 🔓 W0                 | R/O      | false    |
```

Comment:
The Guntamatic heating system provides this data, having the same typo. The channel name is derived from the data provided from heating system.

<img width="371" height="843" alt="image" src="https://github.com/user-attachments/assets/b4ccdde3-4a04-4b6c-96bc-9d1421e95979" />

Reply:
It is not intentionally to change the channel id, will revert that. The label however should be changed. As that is not directly related to the api (its openHAB domain) and used in the UI that should not have sloppy labels due to manufacturer mistakes.

Reply:
`String label = param[0].replace("C02", "CO2").replace("Boiler´s", "Boiler's").replace("Interuption", "Interruption");`

Reply:
Add the above replacements and then we can keep all the proposed changes from this pull-request.
This is not the first channel that is corrected by the binding implementation (`C02` -> `CO2`).